test/io/perun_test.clj

```diff
@@ -77,25 +79,25 @@
-        (value-test :path "test-image.jpg"  :value-fn #(meta-contains? % :io.perun/trace :io.perun/images-dimensions))
-        (value-test :path "test-image.jpeg" :value-fn #(meta= % :width 54))
-        (value-test :path "test-image.jpeg" :value-fn #(meta= % :height 180))
-        (value-test :path "test-image.jpeg" :value-fn #(meta-contains? % :io.perun/trace :io.perun/images-dimensions))
-        (value-test :path "test-image.png"  :value-fn #(meta= % :width 76))
-        (value-test :path "test-image.png"  :value-fn #(meta= % :height 37))
-        (value-test :path "test-image.png"  :value-fn #(meta-contains? % :io.perun/trace :io.perun/images-dimensions))))
+        (value-test :path "test-image.jpg"  :value-fn #(meta= %1 %2 :width 10))
+        (value-test :path "test-image.jpg"  :value-fn #(meta= %1 %2 :height 10))
+        (value-test :path "test-image.jpg"  :value-fn #(meta-contains? %1 %2 :io.perun/trace :io.perun/images-dimensions))
+        (value-test :path "test-image.jpeg" :value-fn #(meta= %1 %2 :width 54))
+        (value-test :path "test-image.jpeg" :value-fn #(meta= %1 %2 :height 180))
+        (value-test :path "test-image.jpeg" :value-fn #(meta-contains? %1 %2 :io.perun/trace :io.perun/images-dimensions))
+        (value-test :path "test-image.png"  :value-fn #(meta= %1 %2 :width 76))
+        (value-test :path "test-image.png"  :value-fn #(meta= %1 %2 :height 37))
+        (value-test :path "test-image.png"  :value-fn #(meta-contains? %1 %2 :io.perun/trace :io.perun/images-dimensions))))
 
 (deftesttask images-resize-test []
   (comp (add-image :path "test-image.jpg" :type "JPG" :width 10 :height 10)
         (p/images-resize :resolutions #{100 200})
-        (value-test :path "public/test-image_100.jpg" :value-fn #(meta= % :width 100))
-        (value-test :path "public/test-image_100.jpg" :value-fn #(meta= % :height 100))
-        (value-test :path "public/test-image_100.jpg" :value-fn #(meta-contains? % :io.perun/trace :io.perun/images-resize))
-        (value-test :path "public/test-image_200.jpg" :value-fn #(meta= % :width 200))
-        (value-test :path "public/test-image_200.jpg" :value-fn #(meta= % :height 200))
-        (value-test :path "public/test-image_200.jpg" :value-fn #(meta-contains? % :io.perun/trace :io.perun/images-resize))))
+        (value-test :path "public/test-image_100.jpg" :value-fn #(meta= %1 %2 :width 100))
+        (value-test :path "public/test-image_100.jpg" :value-fn #(meta= %1 %2 :height 100))
+        (value-test :path "public/test-image_100.jpg" :value-fn #(meta-contains? %1 %2 :io.perun/trace :io.perun/images-resize))
+        (value-test :path "public/test-image_200.jpg" :value-fn #(meta= %1 %2 :width 200))
+        (value-test :path "public/test-image_200.jpg" :value-fn #(meta= %1 %2 :height 200))
+        (value-test :path "public/test-image_200.jpg" :value-fn #(meta-contains? %1 %2 :io.perun/trace :io.perun/images-resize))))
 
 (deftask add-txt-file
```

Comment:
It looks like a lot of these would be generally useful for testing.  We might consider moving them to a testing library so that other related libraries can use them.